src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -47,27 +55,40 @@ public String getTimeAndDate() {
 		return localTimeAndDate;
 	}
 
-	public String changeTheFirstLetterToUppercase(String word) {
-
-		String[] regexResults = word.split("_");
+	public void getDataFromFile() {
 
+		File file = new File("D:/java/europe_capitals.txt");
```

Comment:
Ścieżka do pliku nie dość, że zaszyta na sztywno w kodzie, to jeszcze absolutna. 

Ścieżki do plików powinny być raczej umieszczane w plikach konfiguracyjnych i dopiero stamtąd zaciągane.

Reply:
OK, mógłbyś wyjaśnić, czy te pliki, to są pliki które już istnieją w projekcie i zostały utworzone, przy tworzeniu projektu przez IDE, czy to jest plik który ja stworzę i tam podam ścieżkę do pliku z danymi?

Reply:
Mówię o plikach z danymi, a nie o plikach z kodem.

Reply:
Tutaj też należy zmienić wczytywanie ścieżki do pliku, chcielibyśmy mieć ścieżkę zapisaną 2 pliku konfiguracyjnym (o rozszerzeniu `proprties` lub `yaml`), zaś do beana wstrzykiwać ją za pośrednictwem anotacji `@Value`.

Reply:
Poprawiłem.